survey/survey-service-offering_instance.json

```diff
@@ -0,0 +1,123 @@
+
+{
+       "@context":{
+          "xsd":"http://www.w3.org/2001/XMLSchema#",
+          "survey":"https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/survey/",
+          "gx": "https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#",
+          "general": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/"
+        },
+       "@id":"did:example:survey_service_offering_0xy5vhw3heyjqb4d347hc9zrkw",
+       "@type":"survey:SurveyServiceOffering",
+       "survey:surveyUrl":{
+          "@value":"undefined?session=0xy5vhw3heyjqb4d347hc9zrkw",
+          "@type":"xsd:string"
+       },
+       "survey:surveyCreationTime":{
+          "@value":"2023-08-01T15:14:04",
+          "@type":"xsd:dateTime"
+       },
+       "survey:surveyEndTime":{
+          "@value":"2023-08-02T17:14:00",
+          "@type":"xsd:dateTime"
+       },
+       "gx:description":{
+          "@value":"asdwescxvf",
```

Comment:
can you find a better description and name below?
Or even better: reuse the name and description in GeneralShape